tests/nordic_softdevice/tests/01-run.py

```diff
@@ -1,14 +1,17 @@
 #!/usr/bin/env python3
 
 import sys
+import time
 from testrunner import run
 
 
 def testfunc(child):
-    child.expect("All up, running the shell now")
+    child.expect_exact("All up, running the shell now")
+    child.expect_exact(">")
```

Comment:
Can we expect this on every platform?

Reply:
> Can we expect this on every platform?

What do you mean ?

Reply:
Is `>` shown on every board? I'm not sure, that's why I'm asking.

Reply:
Nordic SoftDevice can only work on nrf52832 based boards so the number of supported platform is quite limited.

`>` is displayed by the shell, a normally hardware agnostic library. I don't see why some platform won't show it.

Reply:
OK :-)